NSURLSessionCancellationTests/NSURLSessionCancellationTests.swift

```diff
@@ -9,27 +9,33 @@
 import XCTest
 
 class NSURLSessionCancellationTests: XCTestCase {
-    
+
     override func setUp() {
```

Comment:
I can sure. I kept them as a remainder of work to do but can be removed for sure.